src/test/java/io/pravega/connectors/hadoop/PravegaInputFormatITCase.java

```diff
@@ -112,6 +115,23 @@ public void testGetSplits() throws IOException, ExecutionException, InterruptedE
         Assert.assertEquals(USED_SPACE_PER_INTEGER * 20 * 3, getTotalUsedSpace(splits));
     }
 
+    @Test
+    public void testGetSplitsWithConfigBuilder() throws IOException, ExecutionException, InterruptedException {
```

Comment:
I think that we should test the builder in isolation.  In other words, no need to create an actual input format; just test that the builder produced the correct configuration entries.   Otherwise we get the redundancy as seen below (`testGetSplits` vs `testGetSplitsWithConfigBuilder`).

Reply:
The builder is already tested separately in PravegaInputFormatTest.java. Remove this part.